Overview level (#2139)
giantswarm/docs: src/content/overview/architecture/_index.md

---
title: Architecture

> description: High overview about how our platform is built and how you can interact with it.

Reviewer: ```suggestion
description: High level technical overview of our platform, its components and how you can interact with it.
```

Author: yeah that is a good point, let's be consistent
